internal/v4/status.go

```diff
@@ -78,6 +79,29 @@ func (h *Handler) checkEntities() (key string, result debugstatus.CheckResult) {
 	return "entities", result
 }
 
+func (h *Handler) checkBaseEntities() (key string, result debugstatus.CheckResult) {
+	resultKey := "base_entities"
+	result.Name = "Base entities in charm store"
+
+	// Retrieve the number of base entities.
+	baseNum, err := h.store.DB.BaseEntities().Count()
+	if err != nil {
+		result.Value = "Cannot count base entities: " + err.Error()
+		return resultKey, result
+	}
+
+	// Retrieve the number of entities.
+	num, err := h.store.DB.Entities().Count()
+	if err != nil {
+		result.Value = "Cannot count entities: " + err.Error()
+		return resultKey, result
+	}
+
+	result.Value = fmt.Sprintf("count: %d", baseNum)
```

Comment:
No need for "count:" part, we it will be used as "Value" : number key-pair, which is easier to read. Applies to the tests as well.